src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -26,14 +26,13 @@ public class AddCommand extends Command {
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
-            + "[" + PREFIX_TAG + "TAG]... "
-            + "[" + PREFIX_TAG + "COMMENT]...\n"
+            + PREFIX_TELEGRAM_HANDLE + "ADDRESS "
+            + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
+            + PREFIX_TELEGRAM_HANDLE + "johndoeisluna"
```

Comment:
There seems to be a missing trailing whitespace here - the telegram handle is not being separated from the prefix tags, which makes it messy.